.gitignore

```diff
@@ -24,3 +24,4 @@ _testmain.go
 *.prof
 
 coverage.txt
+.idea
```

Comment:
this should not go to the per-project `.gitignore`, but rather to your local `.gitignore` which is not tied to any project